lib/commands/general.js

```diff
@@ -6,6 +6,10 @@ commands.mobilePerformEditorAction = async function (opts = {}) {
   return await this.espresso.jwproxy.command('/appium/device/perform_editor_action', 'POST', {action});
 };
 
+commands.mobileSwipe = async function (opts = {}) {
+  const {direction, element} = opts;
```

Comment:
should we unwrap the element first like we do in other drivers?

Reply:
Is there an example of this. The only one I could find was in `appium-uiautomator2-driver`

```javascript
commands.mobileScrollBackTo = async function (opts) {
  const {elementId, elementToId} = opts;
  return await this.uiautomator2.jwproxy.command(`/appium/element/${elementId}/scroll_to/${elementToId}`, 'POST', {});
};
```

Reply:
Very often I see that https://github.com/appium/appium-support/blob/db17f4f827c4e5e53b03481ae428cfcae25e9ff5/lib/util.js#L113 is used for such purpose. @imurchie Can you please provide more details on where this helper should be used?